src/main/java/entities/custom_game/EditorTile.java

```diff
@@ -11,33 +13,19 @@
  * EditorTiles are also stored in TempMaze for conversion to a text file (storage)
  */
 public class EditorTile extends JLabel {
-    private final int X;
-    private final int Y;
     private String name;
     private int numCode;
-
-    private static final String[] secondaryMenuItems = {"photons", "key", "enemy", "start", "end"};
+    private static final String[] secondaryMenuItems = {"photons", "key", "stationaryEnemy", "chasingEnemy", "start", "end"};
```

Comment:
In my most recent PR, I changed all mentions of "Photons" to "Oxygen" and added a new .png file.